eth/utils/transactions.py

```diff
@@ -43,7 +47,10 @@ def extract_signature_v(v: int) -> int:
         return V_OFFSET
 
 
-def create_transaction_signature(unsigned_txn, private_key, chain_id=None):
+def create_transaction_signature(unsigned_txn: BaseTransaction,
+                                 private_key: datatypes.PrivateKey,
+                                 chain_id: int=None) -> Tuple[int, int, int]:
```

Comment:
candidate for eth-typing `VRS = NewType("VRS", Tuple[int, int, int])`

also, `UINT256 = NewType("UINT256", int)` for these values that are specifically bounded within the 256 bit integer space.

Reply:
So `VRS` would then be a `VRS = NewType("VRS", Tuple[UINT256, UINT256, UINT256])`?

Reply:
Yes, though `v` *might* have a lower ceiling than the others (not sure what it is, might be a single byte).

Reply:
Don't talk low level math with me! What does `v` even stand for? Vitalik? ;)

Reply:
https://bitcoin.stackexchange.com/questions/38351/ecdsa-v-r-s-what-is-v/38909

`v` is the recovery id and looks to be reasonably bounded within a single byte (2**8 or `UINT8`)